config/gpu.yml

```diff
@@ -0,0 +1,80 @@
+---
+# This name is used for the Heat stack and as a prefix for the
+# cluster node hostnames.
+cluster_name: gpu
+
+# This parameter should be set to the name of an RSA keypair you have
+# uplaoded to OpenStack.
+cluster_keypair: wendy
+
+# Site-specific network configuration.
+cluster_net:
+  - { net: "ilab", subnet: "ilab" }
+  - { net: "p3-lln", subnet: "p3-lln" }
+  - { net: "p3-bdn", subnet: "p3-bdn" }
+
+# Default nodenet to use for all instances in the cluster
+cluster_nodenet_resource: "Cluster::NodeNet3"
+
+# Enable the use of config drive, for managing IP assignment on IPoIB
+cluster_config_drive: true
+
+# Multi-node application topology.  In this case we have a SLURM
+# deployment formed from a login/controller node and a number of
```

Comment:
Where is the login/controller node defined?

Reply:
Oops this was accidentally left over from the config it was replicated from.